deploy/kubernetes/Makefile

```diff
@@ -1,7 +1,9 @@
 ALL=filebeat metricbeat
 BEAT_VERSION=$(shell head -n 1 ../../libbeat/docs/version.asciidoc | cut -c 17- )
 
-all: ${ALL:=-kubernetes.yaml}
+.PHONY: $(ALL)
```

Comment:
Shoudn't `all` also be a phony target? In our other makefiles, we just make almost all targets `.PHONY`.

Reply:
It worked without it on my machine, but I just pushed it in, just in case it's needed for other systems